frontend/src/pages/cohorts/CohortsPage.tsx

```diff
@@ -123,7 +123,6 @@ export default function CohortsPage() {
           {
             hasCohortCompleteCohortCompletesOptions2: {
               sort: [{ date: SortDirection.Desc }],
-              limit: 1,
```

Comment:
Removed this limit so that the client side (browser) can access all of the `CohortComplete` nodes for a given `Cohort` node.